go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -879,6 +888,48 @@ func (c *TabletConfig) Verify() error {
 	return nil
 }
 
+// verifyUnmanagedTabletConfig checks unmanaged tablet related config for sanity
+func (c *TabletConfig) verifyUnmanagedTabletConfig() error {
+	isUnmanaged := c.UnmanagedTablet
+
+	// Skip checks if tablet is not unmanaged
+	if !isUnmanaged {
+		return nil
+	}
+
+	// Throw error if both host and socket are null
+	if !c.DB.HasGlobalSettings() {
+		return errors.New("no connection parameters specified but unmanaged mode specified")
+	}
+	if c.DB.App.User == "" {
+		return errors.New("database app user not specified")
+	}
+	if c.DB.App.Password == "" {
+		return errors.New("database app user password not specified")
+	}
```

Comment:
I can only think of one more verification to be done for all unmanaged tablets. They should have disable-active-reparents set to true. So something like -
```go
if !mysqlctl.DisableActiveReparents {
	return errors.New("fixing replication should be disabled on unmanaged tablets")
}
```

Reply:
added this validation.